Assets/Scripts/HostSession.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine.UI;

```

Comment:
Ah good idea! There's a callback for room creation failed and I'm sure that will be called when a room fails to create because its name already exists.